### ISSUE TITLE
btcposbal2csv.py doesn't extract segwit

### ISSUE THREAD

**Issue by Jolly-Pirate** [graymauser/btcposbal2csv#15]
btcposbal2csv.py doesn't extract segwit addresses, for example:

Public Address bc1 P2WPKH: bc1qetrqrygl7t0fn6hrdmuug3rrp8vjmum8fd8rsk
Public Address bc1 P2WSH: bc1q26swzzjmqgwvcjxplktpyj9xpkzpv8kgs7t545ku6s7v6fl0xt3slnx8k9

**Comment by graymauser** [comment]
Yes, currently it does not support segwit address extraction. I'm not planning to add support at the moment. I would accept PR with the support.

**Comment by who-biz** [comment]
Hi, I've added support for bech32 addresses in a repository located here: https://github.com/who-biz/chipsposbal2csv.

It has been modified to generate addresses with a different network byte, but the changes are easily portable if @Jolly-Pirate is still looking for a solution.

That repo is now compatible with P2WPKH & P2WSH txs

**Comment by who-biz** [comment]
PR'ed changes for BTC network.
